### PR TITLE
new translation work on 1020

### PR DESCRIPTION
**Pull request by HENGRui6**
[oneflow.selu](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/activation.py#L395)
93643626f59facbddb060860317374c

[oneflow.sigmoid](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/activation.py#L322) 
e57f772527251c93ba3831f0289c487

[oneflow.silu](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/activation.py#L223) 
6ccaf7f4a2754028e96080adc4bce5e

[oneflow.softmax](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/activation.py#L448)
f175d04c3010ab639d2f5c10d10d13a

[oneflow.softsign](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/activation.py#L193)
91ad6bfa6c408054901d033906321a4

[oneflow.sort](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/sort.py#L48)
d2f27a9f82f5c4db85af8df3dc89818

[oneflow.split](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/split.py#L48) 
4277c2e51674e4f56ee0c559ac22ef1

[oneflow.sqrt](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/math_ops.py#L498) 
7b14758ad478dd1ed4263a8a36b8838

[oneflow.squeeze](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/array_ops.py#L274)
501a11f3af9991bd428aa1464a4a3c7

[oneflow.scatter](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/scatter.py#L25) 
55e4f882e13605821c838149abc7095

[oneflow.catter_add](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/scatter.py#L72) 
cf7a2f5689c6c00bbcf330fbb244c7a

[oneflow.scatter_nd](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/scatter.py#L116) 
49de2cd7a1bc3afc7a6b7cba9cee4de

[oneflow.tile](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/tile.py#L24) 
708d59918b3ab5e72590fa3c33c5747

[oneflow.to](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/to.py#L141) 
722b3aee3683ce3ebe7cc282e81b94e

[oneflow.transpose](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/array_ops.py#L216) 
fabbe441889719db184f46ab13d128a

[oneflow.unsqueeze](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/unsqueeze.py#L22) 
2d73702f21c5f67c4b076d2861edf61

[oneflow.var](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/math_ops.py#L1159) 
8611bd9e75748e5d3300d6d2c9b5bb2

[oneflow.view](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/reshape.py#L101)
1750885fecce092cf0192ee5d89b213

[oneflow.where](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/where.py#L22)
bb84f0a9745f38c41ea207b16ad4869

[oneflow.zeros](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/constant.py#L174)
06cbce1e6b76045d1ee01bbbdcfd9f2

[oneflow.relu](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/activation.py#L52)
cc5462911e0b1984e601ed412fdd44d

[oneflow.env.get_rank](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/env.py#L35)
2a7bf54692700da2c5348310832ffb0

[oneflow.sum](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/reduce_ops.py#L130)
b5f80f9a23490ab891b3ec69ea19402

[oneflow.stack](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/array_ops.py#L243)
9ce18f271aeab9052510d880680ea90

[oneflow.sin](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/framework/docstr/math_ops.py#L529)
e477e86bbbae6a20ae3819326f15ffa

[oneflow.slice](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/slice.py#L23)
48da6e5935e26cdc7ce40d5025d429c

[oneflow.slice_update](https://github.com/Oneflow-Inc/oneflow/blob/f053d5c2fdf3e46a11541708d73b4a4284b04e00/python/oneflow/nn/modules/slice.py#L48)
51b0a04adc3c08a465fc9e0b29e4e38

### CODE REVIEW

File: docs/source/cn/__init__.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from .math_ops import *
 from .activation import *
+from .scatter import *
```
[the remark this answers is not in the document]
Comment (doombeaker) [reply docs/source/cn/__init__.py:3]: 不用在 这个PR里做。一个PR集中最一样事情，这样比较清晰，也方便之后其它参与的人参考。

你可以等这个PR合并后，做下这个事情。

File: docs/source/cn/reshape.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      \text{stride}[i] = \text{stride}[i+1] \times \text{size}[i+1]
+
+    
+    否则将无法以视图查看  为形状 :attr:`shape` 且不复制 :attr:`input` （例如通过 :meth:`contiguous`）。
+    当不清楚是否可以执行 :meth:`view` 时，建议使用 :meth:`reshape` ，因为 :meth:`reshape` 在兼容的时候返回
+    :attr:`input` 的视图，不兼容的时候复制 :attr:`input` （相当于调用 :meth:`contiguous` ）。
+
+    参数：
+        - **input** (Tensor)
+        - ***shape** (flow.Size 或 int...)
```
Comment (doombeaker) [inline docs/source/cn/reshape.py:28]: 多了个星号
```suggestion
        - **shape** (flow.Size 或 int...)
```

File: docs/source/cn/scatter.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                [ 2.,  2.,  2.,  2.,  2.]], dtype=oneflow.float32)
+
+    """
+)
+
+reset_docstr(
+    oneflow.scatter_nd,
+    r"""oneflow.scatter_nd(index, update, shape) -> Tensor
+
+    依据 `shape` 创建一个新的元素皆为 0 的 tensor ，并根据 `index` 在新的 tensor 中插入 `update` 的元素。
```
Comment (doombeaker) [inline docs/source/cn/scatter.py:94]: 我们的这个是对齐了 TF 的，加一句与TF的接口一致吧：https://www.tensorflow.org/api_docs/python/tf/scatter_nd

这个接口还是挺复杂的，光看我们现有的说明，应该是无法从无到有知道如何用的。在这个PR里不用解决如何丰富他的问题了，但是可以备忘下，看以后是否有必要把TF的文档详细的部分搬运过来。（说没有必要是因为这个接口用户不一定多，可能会随着时间的推移就没有用户了）

**Review by doombeaker — APPROVED** [review]

File: docs/source/cn/activation.py [hunk 4]
```diff
@@ -100,3 +100,41 @@
...
+        out = \frac{e^x-e^{-x}}{e^x+e^{-x}}
+
+    更多信息请参考 :class:`~oneflow.nn.Tanh` 。
+    """,
+)
+reset_docstr(
+    oneflow.relu,
+    r"""relu(input, inplace) -> Tensor
+
+    对 :attr:`input` 逐元素应用线性整流函数。更多信息请参考 :class:`~oneflow.nn.ReLU` 。
```
[the remark this answers is not in the document]
Comment (HENGRui6) [reply docs/source/cn/activation.py:122]: 明白
